src/js/contexts/ThemeContext/theme-context.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { grommet } from 'grommet/themes';
+import { Box, Text, ThemeContext, Grommet } from 'grommet';
+
+const CustomDefaultProps = () => (
```

Comment:
This component name doesn't make sense to me.
`ExternalComponentWithTheme` would make sense to me, feel free to make other suggestions.